src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -2051,13 +2051,6 @@ describe('ReactDOMComponent', () => {
       );
     });
 
-    it('assigns ajaxify (an important internal FB attribute)', function() {
```

Comment:
I'd like to leave the test specifically for `ajaxify` in. So that we don't accidentally break it with something else.